ivy/functional/backends/paddle/layers.py

```diff
@@ -173,6 +173,8 @@ def conv1d_transpose(
     bias: Optional[paddle.Tensor] = None,
     out: Optional[paddle.Tensor] = None,
 ):
+    if data_format != "NWC":
+        raise ValueError("Invalid data format")
```

Comment:
I think "NCW" is also valid, and actually seems to be the default. The sections like `if data_format == "NWC": ...` are all in aid of transforming data that is in NWC format to be in NCW instead (and back again at the end)

Reply:
So should I add a check for NCW?

Reply:
I thought that an error message could make it more clear, maybe I can add it as an else statement to

     if data_format == "NWC"...

Reply:
It just seems to me that allowing `data_format`s other than NWC is intended, I'm not sure what the problem is that you're trying to fix here

Reply:
Well in that case, maybe I am not thinking in the right direction, my bad😅

Reply:
No worries. If you're looking for ways to contribute to Ivy you can check out our [contributing guide](https://unify.ai/docs/ivy/overview/contributing.html), particularly the section on [ToDo list issues](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues) :)

Reply:
Thank you 😊